be/src/Project/Services/Catalog/Catalog.Api/ServicesModels/Categories/UpdateCategoryServiceModel.cs

```diff
@@ -9,8 +10,7 @@ public class UpdateCategoryServiceModel : BaseServiceModel
         public Guid? Id { get; set; }
         public Guid? ParentId { get; set; }
         public string Name { get; set; }
-        public string Schema { get; set; }
-        public string UiSchema { get; set; }
+        public IEnumerable<CategorySchema> Schemas { get; set; }
```

Comment:
**suggestion:** Change to CategorySchemaServiceModel instead of CategorySchema. It is service model. Not Enittiy.